state_machine/applications/flight/lib/radio_utils/memory_buffered_message.py

```diff
@@ -11,7 +12,7 @@ class NaiveMessage(Message):
     :type str: str | bytes | bytearray
     """
 
-    packet_len = 249  # not 251 because for some reason packet loss is extremely high at this threshold
+    packet_len = PACKET_DATA_LEN
```

Comment:
I noticed that this is 251 in the __init__.py. Should it be 249?

Reply:
Max packet length is 256, lora header is 4 bytes, our extra header is 1 byte.
However I found that LORA doesn't seem to work with such large packet sizes, when doing field testing I often reduce the "MAX_PACKET_LENGTH" from 252 to around 200.

I think this is the correct value, we will probably lower MAX_PACKET_LEN however (if we can even get LORA to work)